service/client_state.cc

```diff
@@ -191,6 +192,28 @@ future<> service::client_state::has_access(const sstring& ks, auth::permission p
         }
     }
 
+    if (resource.kind() == auth::resource_kind::data && service::get_local_storage_service().cluster_supports_cdc()) {
```

Comment:
this line is not up-to-date with b0638606e5 - `cluster_supports_cdc` is now a method of `feature_service`, not `storage_service`. Plz rebase

Reply:
Done.